exotica/src/Tools.cpp

```diff
@@ -146,4 +148,47 @@ namespace exotica
       return name;
   }
 
+  std::string parsePath(const std::string& path)
+  {
+      std::string ret = path;
+      std::smatch matches;
+      std::regex_search(ret, matches, std::regex("\\{([^\\}]+)\\}"));
+      for(auto& match : matches)
+      {
+          std::string package = match.str();
+          if(package[0]=='{') continue;
+          std::string package_path = ros::package::getPath(package);
+          if(package_path=="") throw_pretty("Unknown package '"<<package<<"'");
```

Comment:
This is the line that throws on CI. CI uses gcc-4.9 but perhaps there are still issues with the regex implementation in that version.